examples/cra-demo/package.json

```diff
@@ -23,17 +23,16 @@
   },
   "dependencies": {
     "@formspree/react": "*",
-    "react": "^18.2.0",
-    "react-copy-to-clipboard": "^5.1.0",
```

Comment:
Drop `react-copy-to-clipboard` from the example, since its outdated types cause conflicts with React 19.

Use `copy-to-clipboard` instead.